### PR TITLE
Grow 699 lenders teams lightbox

### PR DESCRIPTION
**Pull request by mcstover**

### CODE REVIEW

**Review by BoulderBrains — APPROVED** [review]

File: src/components/BorrowerProfile/LendersAndTeams.vue [hunk 1]
```diff
@@ -55,16 +55,69 @@
...
+							{{ item.lenderPage.whereabouts }}
+						</span>
+					</div>
+				</div>
+			</div>
+
+			<kv-button
+				v-if="items.length < totalItemCount"
+				class="tw-w-full"
+				@click.prevent="loadMore"
```
Comment (ryan-ludwig) [inline src/components/BorrowerProfile/LendersAndTeams.vue:108]: No need to `click.prevent` anymore with KvButton, can just do `click`  :)

File: src/components/BorrowerProfile/LendersAndTeams.vue [hunk 2]
```diff
@@ -55,16 +55,69 @@
...
+				</div>
+			</template>
+
+			<div class="tw-mb-3 tw-grid tw-grid-cols-2 md:tw-grid-cols-3 tw-gap-2">
+				<!-- TODO: Replace with Generic component for showing Either Team or Lender image and information -->
+				<div
+					v-for="(item, index) in items" :key="index"
+					class="tw-flex tw-flex-col md:tw-flex-row md:tw-items-center tw-mb-3"
+				>
+					<kv-loading-placeholder class="tw-rounded md:tw-hidden tw-mb-1.5" :style="{height: '128px'}" />
```
Comment (ryan-ludwig) [inline src/components/BorrowerProfile/LendersAndTeams.vue:92]: Minor, ideally using rems here and below

**Review by ryan-ludwig — APPROVED** [review]
Nice!

File: src/components/BorrowerProfile/LendersAndTeams.vue (hunk 2 above)
Comment (mcstover) [reply src/components/BorrowerProfile/LendersAndTeams.vue:92]: These will go away completely once we have the actual item component in place. The lightbox won't need preview content as it start with the previously loaded and shown team/lender set.

File: src/components/BorrowerProfile/LendersAndTeams.vue (hunk 1 above)
Comment (mcstover) [reply src/components/BorrowerProfile/LendersAndTeams.vue:108]: Cool, will clean up in some follow up

**Pull request merged**